README.rst

```diff
@@ -0,0 +1,138 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-itertools/badge/?version=latest
+    :target: https://circuitpython.readthedocs.io/projects/adafruit_itertools/en/latest/
+    :alt: Documentation Status
+
+.. image:: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+.. image:: https://travis-ci.com/adafruit/Adafruit_CircuitPython_itertools.svg?branch=master
+    :target: https://travis-ci.com/adafruit/Adafruit_CircuitPython_itertools
```

Comment:
The URLs for the Travis badge need to match the repo capitalization. 
```
.. image:: https://travis-ci.com/adafruit/Adafruit_CircuitPython_IterTools.svg?branch=master
    :target: https://travis-ci.com/adafruit/Adafruit_CircuitPython_IterTools
```

Reply:
Done